python/pyspark/ml/pipeline.py

```diff
@@ -146,6 +125,57 @@ class Pipeline(Estimator):
     pipeline stages. If there are no stages, the pipeline acts as an
     identity transformer.
 
+    >>> from pyspark.ml.feature import HashingTF
```

Comment:
A unit test this long should go in tests.py, not in a doctest.  It'd be nice to have a very short, simple doc test as well though.